pep-0693.rst

```diff
@@ -52,11 +52,11 @@ Actual:
 - 3.12.0 alpha 7: Tuesday, 2023-04-04
 - 3.12.0 beta 1: Monday, 2023-05-22
   (No new features beyond this point.)
+- 3.12.0 beta 2: Tuesday, 2023-06-06
+- 3.12.0 beta 3: Tuesday, 2023-06-20
```

Comment:
```suggestion
- 3.12.0 beta 3: Monday, 2023-06-19
```

To match the release date on the release page (https://www.python.org/downloads/release/python-3120b3/) and the day the release was cut.